### PR TITLE
DBZ-1069 Snapshot is completed when offset is recovered

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

**Comment by jpechane** [comment]
https://issues.jboss.org/browse/DBZ-1069

File: debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java [hunk 1]
```diff
@@ -256,6 +257,66 @@ public void streamChangesWhileStopped() throws Exception {
...
+            );
+            connection.execute(
+                    "INSERT INTO tableb VALUES(" + id + ", 'b')"
+            );
+        }
+
+        start(SqlServerConnector.class, config);
+        assertConnectorIsRunning();
+        records = consumeRecordsByTopic(1 + RECORDS_PER_TABLE * TABLES).allRecordsInOrder();
+        records = records.subList(1, records.size());
```
Comment (grzegorz8) [inline debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java]: It seems to be wrong. If you have a look at records, they have ids: 10, 11, 12, ... Should be 100, 101.

Comment (grzegorz8) [reply debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java]: This means that the same records have been fetched twice - during initial load and streaming.

Comment (jpechane) [reply debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java]: You are right, but this is a different issue - I need to think about it more. The last snapshot record should probably contan the LSN from which we should start reading in streaming phase.

**Comment by grzegorz8** [comment]
@jpechane Looks good now :+1:

**Comment by gunnarmorling** [comment]
@grzegorz8, does the PR resolve the original issue for you? I'm a bit unsure given your last comment on the JIRA issue.

**Comment by grzegorz8** [comment]
@gunnarmorling Yes, it does :)

**Comment by gunnarmorling** [comment]
Ok, thanks!

**Pull request merged**